contact.html

```diff
@@ -14,12 +14,16 @@
     <header>
         <h1>Contact Us</h1>
         <nav>
-            <ul>
-                <li><a href="index.html">Home</a></li>
-                <li><a href="index.html#LTS">About <abbr title="The Little Taco Shop">LTS</abbr></a></li>
-                <li><a href="index.html#Menu">Our Menu</a></li>
-                <li><a href="hours.html">Store Hours</a></li>
-            </ul>
+            <table>
```

Comment:
If you use a table for the navigation, it will look much better by adding a little style "<table style="width:100%">. HTML tables does not occupy the full length of a page automatically. 
![Screenshot 2024-07-22 131244](https://github.com/user-attachments/assets/9f75d827-43b3-4d23-9b18-41414bf3415f)

Reply:
This looks better, ill add the changes tomorrow.